packages/cli/lib/cli/cliObjects.js

```diff
@@ -36,13 +36,46 @@ module.exports = class CLIObjects extends CLICommand {
             case 'delete':
             case 'del':
                 return this.delete(args);
+            case 'getVersion':
+                return this.getVersion(args);
+            case 'setVersion':
+                return this.setVersion(args);
             default:
                 CLI.error.unknownCommand('object', command);
                 showHelp();
                 return void callback(3);
         }
     }
 
+    /**
+     * Get the protocol version
+     */
+    getVersion() {
+        const { callback, dbConnect } = this.options;
+        dbConnect(async objects => {
+            const version = await objects.getProtocolVersion();
+            console.log(`Current Objects DB protocol version: ${version}`);
+            return void callback();
+        });
+    }
+
+    /**
+     * Set protocol version
+     */
+    setVersion(args) {
+        const { callback, dbConnect } = this.options;
+        dbConnect(async objects => {
```

Comment:
I think we should add a "really"? question or such because of the restart fact ... we should tell that to the user very explicitely

Reply:
added